src/index.html

```diff
@@ -11,7 +11,151 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+     onsubmit="onSubmit()">
+
+      <fieldset class="container">
+        <legend>Personal information</legend>
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+          type="text"
+          name="surname"
+          id="surname">
```

Comment:
Fix indentations here and all over your project.
Also, closing tag bracket should be on the same level as opening one.
```suggestion
          <input
            type="text"
            name="surname"
            id="surname"
          >
```